src/sentry/static/sentry/less/shared-components.less

```diff
@@ -845,13 +845,13 @@ table.integrations {
 
   .switch-toggle {
     top: 2px;
-    left: 2px;
+    transform: translate(2px, 0);
```

Comment:
translateX is marginally better here because it won't override any potential horizontal translations on the same element, which happens surprisingly often.

Great pull btw! Translations + Animations = 👍👍👍👍👍